### ISSUE TITLE
Check for uses of `--n`

### ISSUE THREAD

**Issue by tjkuson** [astral-sh/ruff#5992]
`unary-prefix-increment` (`B002`) warns users about

```python
++n
```

but not 

```python
--n
```

It would be good if either the rule were changed to check for both, or if a new rule were created to check for unary prefix decrement operators. The rationale is the same; users coming from other languages which support `++` and `--` unary prefix operators might not realise that these 'operators' in Python don't do anything, causing subtle bugs.

**Comment by charliermarsh** [comment]
Fine with me to add to the existing rule.
